examples/cpp/pyperf/PyPerfBPFProgram.cc

```diff
@@ -595,6 +598,7 @@ int read_python_stack(struct pt_regs* ctx) {
   goto submit;
 
 complete:
+  event->kernel_stack_id = kernel_stacks.get_stackid(ctx, BPF_F_REUSE_STACKID);
```

Comment:
I think we want to collect this on Python stack errors as well to ease debugging.

Reply:
Yea, we should. I noted it in one of the commit messages. In a separate PR